worker/src/test/java/org/verapdf/webapp/worker/listener/ExecutableTaskWorkerTests.java

```diff
@@ -0,0 +1,555 @@
+				"/files/report-PDFA_1_A.json").readAllBytes(), StandardCharsets.UTF_8);
+		Mockito.doThrow(new RestClientResponseException("400 : [incorrect parameters]",
+				HttpStatus.BAD_REQUEST.value(), "", null,
+				"incorrect parameters".getBytes(), null))
+				.when(localStorageServiceClient)
+				.saveFile(argThat(new FileAsJsonMatcher(expectedReportAsString)),
+						eq("35a7d1deffc1ad490bcc5eb3c557a399"));
+
+		executableTaskWorker.handleMessage(
+				"{\"jobId\":\"780e3129-42a7-4154-8ce9-436fc1a6dc35\","
+						+ "\"fileId\":\"07b17d5d-1010-4980-b280-94129cb13838\","
+						+ "\"profile\":\"PDFA_1_A\"}");
+
+		String expectedMessage
+				= "{\"jobId\":\"780e3129-42a7-4154-8ce9-436fc1a6dc35\","
+				+ "\"fileId\":\"07b17d5d-1010-4980-b280-94129cb13838\","
+				+ "\"errorType\":\"SAVE_RESULT_FILE_ERROR\","
+				+ "\"errorMessage\":\"incorrect parameters\"}";
+
+		Mockito.verify(localStorageServiceClient).getFileDescriptorById(fileId);
+		Mockito.verify(localStorageServiceClient).getFileResourceById(fileId);
+		Mockito.verify(localStorageServiceClient).saveFile(any(File.class),
+				eq("35a7d1deffc1ad490bcc5eb3c557a399"));
+		Mockito.verify(queueSender).sendMessage(expectedMessage);
+	}
+
+	@Test
+	public void requestTimeoutExceptionOnSaveReportFileOnProcessTest() throws URISyntaxException, IOException {
+		UUID fileId = UUID.fromString("07b17d5d-1010-4980-b280-94129cb13838");
+
+		Mockito.doReturn(createDescriptor(fileId))
+				.when(localStorageServiceClient)
+				.getFileDescriptorById(fileId);
+		Mockito.doReturn(createResource())
+				.when(localStorageServiceClient)
+				.getFileResourceById(fileId);
+		String expectedReportAsString = new String(getClass().getResourceAsStream(
+				"/files/report-PDFA_1_A.json").readAllBytes(), StandardCharsets.UTF_8);
+		Mockito.doThrow(new RestClientResponseException("408 : [request timeout]",
+				HttpStatus.REQUEST_TIMEOUT.value(), "", null, "request timeout".getBytes(), null))
+				.when(localStorageServiceClient)
+				.saveFile(argThat(new FileAsJsonMatcher(expectedReportAsString)),
+						eq("35a7d1deffc1ad490bcc5eb3c557a399"));
+
+		executableTaskWorker.handleMessage(
+				"{\"jobId\":\"780e3129-42a7-4154-8ce9-436fc1a6dc35\","
+						+ "\"fileId\":\"07b17d5d-1010-4980-b280-94129cb13838\","
+						+ "\"profile\":\"PDFA_1_A\"}");
+
+		String expectedMessage
+				= "{\"jobId\":\"780e3129-42a7-4154-8ce9-436fc1a6dc35\","
+				+ "\"fileId\":\"07b17d5d-1010-4980-b280-94129cb13838\","
+				+ "\"errorType\":\"SAVE_RESULT_FILE_ERROR\","
+				+ "\"errorMessage\":\"request timeout\"}";
+
+		Mockito.verify(localStorageServiceClient).getFileDescriptorById(fileId);
+		Mockito.verify(localStorageServiceClient).getFileResourceById(fileId);
+		Mockito.verify(localStorageServiceClient).saveFile(any(File.class),
+				eq("35a7d1deffc1ad490bcc5eb3c557a399"));
+		Mockito.verify(queueSender).sendMessage(expectedMessage);
+	}
+
+	private class FileAsJsonMatcher implements ArgumentMatcher<File>{
```

Comment:
1. all inner classes should be placed after all methods (even private).
2. make it `private static class...` (as it does not require the instance of the main class